docs/generated/.files_generated

```diff
@@ -1,2 +1,6 @@
+artifacts
```

Comment:
@stevekuznetsov the change to BASETMPDIR seems to sneak in these extra directories. I am going to revert it, I don't want to block this PR on pre-existing bash code...